### PR TITLE
Update Comp sci -> the-components-of-a-graph.md

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**
Split insight into 2 smaller ones

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/comp-sci/data-structures-and-algorithms/intro-graphs/the-components-of-a-graph.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/comp-sci/data-structures-and-algorithms/intro-graphs/the-components-of-a-graph.md?ref=610fb0164bf084430b3a88cb2e55403e56125d1e)

File: comp-sci/data-structures-and-algorithms/intro-graphs/the-components-of-a-graph-ii.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+parent: the-graph-data-structure
+
+---
+
+# The Components of a Graph
+
+---
+## Content
+
+A graph is **connected** if between every vertex in the graph, there is a *path*. The graph in the previous insight[1] was connected, while the one below isn't (you cannot get from *A* to *B*, for example):
```
Comment (kapnobatai137) [inline comp-sci/data-structures-and-algorithms/intro-graphs/the-components-of-a-graph-ii.md]: can you guys think of any better way to reference a footnote? @Stefan-Stojanovic  @nem035 
```suggestion
A graph is **connected** if between every vertex in the graph, there is a *path*. The graph in this footnote[1] is connected, while the one below isn't (you cannot get from *A* to *B*, for example):
```

File: comp-sci/data-structures-and-algorithms/intro-graphs/the-components-of-a-graph.md [hunk 2]
```diff
@@ -36,51 +36,4 @@ A *path* is **simple** if it contains each edge and vertex at most *once*. If a
...
-
-If the graph is connected as well, it is called a **tree**. A **spanning tree** is a *spanning subgraph* (a subgraph containing all of the graph's nodes) that is a tree. As the tree below contains all nodes in the initial graph, it is a spanning tree:
-
-![spanningtree](https://img.enkipro.com/48d9282e1686cda71848ec61545ed099.png)
-
-
-Note: The notion of **tree** here is not related to the one of the *Tree Data Type* since it *does not need* to have a root.
-
----
-## Practice
```
Comment (kapnobatai137) [inline comp-sci/data-structures-and-algorithms/intro-graphs/the-components-of-a-graph.md]: we could add a  here like this:

What is a cycle?

???

* good answer
* bad answer

File: comp-sci/data-structures-and-algorithms/intro-graphs/the-components-of-a-graph-ii.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+  - introduction
+
+  - workout
+
+parent: the-graph-data-structure
+
+---
+
+# The Components of a Graph
```
Comment (kapnobatai137) [inline comp-sci/data-structures-and-algorithms/intro-graphs/the-components-of-a-graph-ii.md]: how is this?
```suggestion
# More Components of a Graph
```

**Pull request merged**